spec/features/user_sign_in_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe "User login", type: :feature do
+  let(:password) { Faker::Internet.password }
+  let(:user) { create(:user, password: password) }
+
+  scenario "with valid email and password" do
+    sign_in_with(user.email, password)
+    expect(page).to have_content(t("activerecord.session.form.flash.login_success"))
+  end
+
+  scenario "with invalid email" do
+    sign_in_with(Faker::Internet.email, password)
+    expect(page).to have_content(t("activerecord.session.form.flash.login_error"))
+  end
+
+  scenario "with invalid password" do
+    sign_in_with(user.email, Faker::Internet.password)
+    expect(page).to have_content(t("activerecord.session.form.flash.login_error"))
+  end
+
+  it "when user log out" do
```

Comment:
If you are doing acceptance tests with Capybara you should use "scenario" here too, instead of "it"